client/extensions/tga-sign-off/src/components/SignOffListItem.tsx

```diff
@@ -34,6 +34,14 @@ type IProps = IPropsApproved | IPropsPendingOrExpired | IPropsNotSend;
 export function SignOffListItem(props: IProps) {
     const {formatDateTime, gettext} = superdesk.localization;
 
+    const isExpired = (date: string) => {
```

Comment:
move this one out of `SignOffListItem` function so it is clear it is not using anything from parent scope.